src/IonTypes.ts

```diff
@@ -14,9 +14,11 @@
 
 import { IonType } from "./IonType";
 
+/** Enumeration of the Ion types. */
 export const IonTypes = {
   NULL      : new IonType(  0, "null",       true,   false, false,  false ),
   BOOL      : new IonType(  1, "bool",       true,   false, false,  false ),
+  // note that INT is actually 0x2 **and** 0x3 in the binary type
```

Comment:
Nit: in the binary ~~type~~ format/encoding

Reply:
Fixed.